### PR TITLE
[CINN] Remove cinn_bucket_compile flag

### PR DESCRIPTION
**Pull request by Dmovic**
### PR Category

CINN

### PR Types

Improvements

### Description

Remove cinn_bucket_compile flag

Pcard-67164

### CODE REVIEW

**Review by Hongqing-work — DISMISSED** [review]

File: paddle/cinn/backends/codegen_cuda_host.h [hunk 1]
```diff
@@ -77,8 +72,6 @@ class CodeGenGpuHost : public CodeGenHost {
    * external function.
    *
    */
-  llvm::Value *LowerGPUKernelLauncher(const ir::_LoweredFunc_ *func);
```
Comment (zyfncg) [inline paddle/cinn/backends/codegen_cuda_host.h:80]: 上面的注释也可以删了

**Review by zyfncg — DISMISSED** [review]

File: paddle/cinn/backends/codegen_cuda_host.h (hunk 1 above)
Comment (Dmovic) [reply paddle/cinn/backends/codegen_cuda_host.h:80]: Done, thanks.

**Review by zyfncg — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by zhangbo9674 — APPROVED** [review]
LGTM

**Pull request merged**